### PR TITLE
Update docs related to Resources menu change

### PR DESCRIPTION
**Pull request by catherineluse**
This PR addresses this issue https://github.com/rancher/docs/issues/1655

### CODE REVIEW

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ If you want to create HPAs that scale based on other metrics than CPU and memory
 
 1. From the **Global** view, open the project that you want to deploy a HPA to.
 
-1. Select **Workloads** in the navigation bar and then select the **HPA** tab.
+1. Click **Resources > HPA.** In versions prior to v2.3.0, select **Workloads** in the navigation bar and then select the **HPA** tab.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md]: HPA was added in 2.3.0 so the reference to earlier Rancher versions can be removed.

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md [hunk 2]
```diff
@@ -29,13 +29,13 @@ If you want to create HPAs that scale based on other metrics than CPU and memory
 
 1. Click **Create** to create the HPA.
 
-> **Result:** The HPA is deployed to the chosen namespace. You can view the HPA's status from the project's Workloads > HPA view.
+> **Result:** The HPA is deployed to the chosen namespace. You can view the HPA's status from the project's Resources > HPA view. (In versions prior to v2.3.0, you can view the HPA status from the Workloads > HPA view.)
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md]: HPA was added in 2.3.0 so the reference to earlier Rancher versions can be removed.

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md [hunk 3]
```diff
@@ -29,13 +29,13 @@ If you want to create HPAs that scale based on other metrics than CPU and memory
...
 
 ## Get HPA Metrics and Status
 
 1. From the **Global** view, open the project with the HPAs you want to look at.
 
-1. Select **Workloads** in the navigation bar and then select the **HPA** tab. The **HPA** tab shows the number of current replicas.
+1. Click **Resources > HPA.** (In versions prior to v2.3.0, select **Workloads > HPA.**) The **HPA** tab shows the number of current replicas.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md]: HPA was added in 2.3.0 so the reference to earlier Rancher versions can be removed.

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ If you want to create HPAs that scale based on other metrics than CPU and memory
 
 1. From the **Global** view, open the project that you want to delete an HPA from.
 
-1. Select **Workloads** in the navigation bar and then select the **HPA** tab.
+1. Click **Resources > HPA.** (In versions prior to v2.3.0, select **Workloads > HPA.**)
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md]: HPA was added in 2.3.0 so the reference to earlier Rancher versions can be removed.

File: content/rancher/v2.x/en/k8s-in-rancher/load-balancers-and-ingress/ingress/_index.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ Ingress can be added for workloads to provide load balancing, SSL termination an
 
 1. From the **Global** view, open the project that you want to add ingress to.
 
-1. Select the **Load Balancing** tab. Then click **Add Ingress**.
+1. Click **Resources > Workloads > Load Balancing.** (In versions prior to v2.3.0, click the **Load Balancing** tab.) Then click **Add Ingress**.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/k8s-in-rancher/load-balancers-and-ingress/ingress/_index.md]: Update to 
``` 
Click **Resources** in the main navigation bar. Click the **Load Balancing** tab. Then click **Add Ingress**.
```

Before the Load Balancing tab was under the Workloads in the main , so the previous instructions weren't clear.

File: content/rancher/v2.x/en/k8s-in-rancher/service-discovery/_index.md [hunk 6]
```diff
@@ -12,7 +12,7 @@ However, you also have the option of creating additional Service Discovery recor
 
 1. From the **Global** view, open the project that you want to add a DNS record to.
 
-1. Select the **Service Discovery** tab. Then click **Add Record**.
+1. Click **Resources > Workloads > Service Discovery.** (In versions prior to v2.3.0, click the **Service Discovery** tab.) Then click **Add Record**.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/k8s-in-rancher/service-discovery/_index.md]: Update to 
``` 
Click **Resources** in the main navigation bar. Click the **Service Discovery** tab. Then click **Add Record**.
```

Before the Service Discovery tab was under the Workloads in the main , so the previous instructions weren't clear.

File: content/rancher/v2.x/en/v1.6-migration/discover-services/_index.md [hunk 7]
```diff
@@ -62,7 +62,7 @@ In the image below, the `web-deployment.yml` and `web-service.yml` files [create
 
 Just as you can create an alias for Rancher v1.6 services, you can do the same for Rancher v2.x workloads. Similarly, you can also create DNS records pointing to services running externally, using either their hostname or IP address. These DNS records are Kubernetes service objects.
 
-Using the v2.x UI, use the context menu to navigate to the `Project` view and choose the **Service Discovery** tab. All existing DNS records created for your workloads are listed under each namespace.
+Using the v2.x UI, use the context menu to navigate to the `Project` view. Then click **Resources > Workloads > Service Discovery.** (In versions prior to v2.3.0, click the **Service Discovery** tab.) All existing DNS records created for your workloads are listed under each namespace.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/v1.6-migration/discover-services/_index.md]:
```
(In versions prior to v2.3.0, click the **Workloads -> Service Discovery** tab.)
```

File: content/rancher/v2.x/en/v1.6-migration/load-balancing/_index.md [hunk 8]
```diff
@@ -62,12 +62,12 @@ Although Rancher v2.x supports HTTP and HTTPS hostname and path-based load balan
...
+You can launch a new load balancer to replace your load balancer from v1.6. Using the Rancher v2.x UI, browse to the applicable project and choose **Resources > Workloads > Load Balancing.** (In versions prior to v2.3.0, click **Workloads > Load Balancing.**) Then click **Deploy**. During deployment, you can choose a target project or namespace.
 
 >**Prerequisite:** Before deploying Ingress, you must have a workload deployed that's running a scale of two or more pods.
 >
 
-For balancing between these two pods, you must create a Kubernetes Ingress rule. To create this rule, navigate to your cluster and project, and then select the **Load Balancing** tab. Then click **Add Ingress**. This GIF below depicts how to add Ingress to one of your projects.
+For balancing between these two pods, you must create a Kubernetes Ingress rule. To create this rule, navigate to your cluster and project, and click **Resources > Workloads > Load Balancing.** (In versions prior to v2.3.0, click the **Load Balancing** tab.) Then click **Add Ingress**. This GIF below depicts how to add Ingress to one of your projects.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/v1.6-migration/load-balancing/_index.md]:
```
(In versions prior to v2.3.0, click **Workloads > Load Balancing.**)```

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md (hunk 1 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md]: removed reference to earlier versions

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md (hunk 2 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md]: removed references to earlier versions

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md (hunk 3 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md]: removed references to earlier versions

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md (hunk 4 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui/_index.md]: removed reference to earlier versions

File: content/rancher/v2.x/en/k8s-in-rancher/load-balancers-and-ingress/ingress/_index.md (hunk 5 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/k8s-in-rancher/load-balancers-and-ingress/ingress/_index.md]: replaced step

File: content/rancher/v2.x/en/k8s-in-rancher/service-discovery/_index.md (hunk 6 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/k8s-in-rancher/service-discovery/_index.md]: replaced step

File: content/rancher/v2.x/en/v1.6-migration/discover-services/_index.md (hunk 7 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/v1.6-migration/discover-services/_index.md]: updated

File: content/rancher/v2.x/en/v1.6-migration/load-balancing/_index.md (hunk 8 above)
Comment (catherineluse) [reply content/rancher/v2.x/en/v1.6-migration/load-balancing/_index.md]: updated

File: content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/_index.md [hunk 9]
```diff
@@ -18,8 +18,8 @@ The way that you manage HPAs is different based on your version of the Kubernete
...
+- **For Rancher Prior to v2.3.0:** To manage and configure HPAs, you need to use `kubectl`. For instructions on how to create, manage, and scale HPAs, refer to [Managing HPAs with kubectl]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-kubectl).
+- **For Rancher v2.3.0**: You can create, manage, and delete HPAs using the Rancher UI. From the Rancher UI you can configure the HPA to scale based on CPU and memory utilization. For more information, refer to [Managing HPAs with the Rancher UI]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-rancher-ui). To scale the HPA based on custom metrics, you still need to use `kubectl`. For more information, refer to [Configuring HPA to Scale Using Custom Metrics with Prometheus]({{< baseurl >}}/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/manage-hpa-with-kubectl/#configuring-hpa-to-scale-using-custom-metrics-with-prometheus).
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/_index.md]: Let's put the Rancher v2.3 at the top. 

And make it `For Rancher v2.3.0+`

Comment (catherineluse) [reply content/rancher/v2.x/en/k8s-in-rancher/horitzontal-pod-autoscaler/_index.md]: Moved up the v2.3.0 section and changed it to v2.3.0+

**Pull request merged**